pip install failed on Ubuntu 22.04.2 LTS / Debian 12.0

Since the 0.1.10 release I'm having installs failing when running `pip3 install`. Initial failures were on a Ubuntu 22.04.2 LTS box, with Python 3.10.6 / pip3 23.1.2 installed within a venv configuration. I've since replicated it on Debian 12 with Python 3.11.2 and Kali 2023.2 also with Python 3.11.2 / pip3 23.0.1. The build will fail both within a venv as well as with an user install with the following trace:

```sh
$ python3 -m venv test       
$ source test/bin/activate
                                                                                                                     
┌──(test)
└─$ pip3 install binary2strings
Collecting binary2strings
  Downloading binary2strings-0.1.10.tar.gz (9.3 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Installing backend dependencies ... done
  Preparing metadata (pyproject.toml) ... done
Building wheels for collected packages: binary2strings
  Building wheel for binary2strings (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for binary2strings (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      running bdist_wheel
      running build
      running build_ext
      x86_64-linux-gnu-gcc -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/home/rokkr/Scripts/test-binary-strings/test/include -I/usr/include/python3.11 -c flagcheck.cpp -o flagcheck.o -std=c++17
      building 'binary2strings' extension
      creating build
      creating build/temp.linux-x86_64-cpython-311
      creating build/temp.linux-x86_64-cpython-311/src
      x86_64-linux-gnu-gcc -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -DVERSION_INFO=0.1.10 -Isrc -I/tmp/pip-build-env-fv5_42_5/overlay/lib/python3.11/site-packages/pybind11/include -I/home/rokkr/Scripts/test-binary-strings/test/include -I/usr/include/python3.11 -c src/binary2strings.cpp -o build/temp.linux-x86_64-cpython-311/src/binary2strings.o -std=c++17 -fvisibility=hidden -g0
      src/binary2strings.cpp:1:10: fatal error: binary2strings.hpp: No such file or directory
          1 | #include "binary2strings.hpp"
            |          ^~~~~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for binary2strings
Failed to build binary2strings
ERROR: Could not build wheels for binary2strings, which is required to install pyproject.toml-based projects
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,1 @@
+include src/*.hpp

--- a/setup.py
+++ b/setup.py
@@ -8,20 +8,24 @@ from pybind11 import get_cmake_dir
 from pybind11.setup_helpers import Pybind11Extension, build_ext
 from setuptools import setup, Extension, find_packages
 
-__version__ = "0.1.12"
+__version__ = "0.1.13"
 
 ext_modules = [
     Pybind11Extension("binary2strings",
         sorted(glob("src/*.cpp")),  # Sort source files for reproducibility
-        headers = sorted(glob("src/*.hpp")), # Critical for Linux package building
         define_macros = [('VERSION_INFO', __version__)],
         include_dirs = ["src"],
         ),
 ]
 
 # Note to self to build and upload skip existing:
+#   Delete dist/ and build/
 #   python setup.py sdist bdist_wheel
 #   twine upload dist/* --skip-existing
+#     Get token from https://pypi.org/manage/account/token/
+#     Use username __token__
+#     Use password from token
+#     Delete token after use
 with open("README.md", "r") as fh:
     long_description = fh.read()
 setup(

--- a/setup.py
+++ b/setup.py
@@ -8,11 +8,12 @@ from pybind11 import get_cmake_dir
 from pybind11.setup_helpers import Pybind11Extension, build_ext
 from setuptools import setup, Extension, find_packages
 
-__version__ = "0.1.11"
+__version__ = "0.1.12"
 
 ext_modules = [
     Pybind11Extension("binary2strings",
         sorted(glob("src/*.cpp")),  # Sort source files for reproducibility
+        headers = sorted(glob("src/*.hpp")), # Critical for Linux package building
         define_macros = [('VERSION_INFO', __version__)],
         include_dirs = ["src"],
         ),

